packages/stream_chat_flutter/lib/src/message_text.dart

```diff
@@ -7,43 +8,45 @@ import 'utils.dart';
 
 class MessageText extends StatelessWidget {
   const MessageText({
-    Key key,
-    @required this.message,
-    @required this.messageTheme,
+    Key? key,
+    required this.message,
+    required this.messageTheme,
     this.onMentionTap,
     this.onLinkTap,
   }) : super(key: key);
 
   final Message message;
-  final void Function(User) onMentionTap;
-  final void Function(String) onLinkTap;
-  final MessageTheme messageTheme;
+  final void Function(User)? onMentionTap;
+  final void Function(String)? onLinkTap;
+  final MessageTheme? messageTheme;
 
   @override
   Widget build(BuildContext context) {
-    final text = _replaceMentions(message.text).replaceAll('\n', '\\\n');
+    final text = _replaceMentions(message.text)!.replaceAll('\n', '\\\n');
```

Comment:
Consider making the _replaceMentions method non nullable and accept a NN string, pass a default value (message.text ?? '') when calling it